streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -246,7 +247,7 @@ private KafkaMetric getMetric(final String nameFormat, final String descriptionF
             String.format(nameFormat, "commit"),
             "stream-task-metrics",
             String.format(descriptionFormat, "commit"),
-            mkMap(mkEntry("task-id", taskId))
+            mkMap(mkEntry("task-id", taskId), mkEntry("client-id", "test"))
```

Comment:
This is required per the docs, but we previously only added it in production code paths. Now we add it in all code paths.